src/wp-includes/functions.php

```diff
@@ -8071,7 +8071,7 @@ function wp_cache_set_last_changed( $group ) {
 	 * @since 6.3.0
 	 *
 	 * @param string    $group         The cache group name.
-	 * @param int       $time          The new last changed time.
+	 * @param float     $time          The new last changed time.
```

Comment:
This was never `int`, it's always been `string`: `0.91192600 1743003204`

Reply:
The `wp_cache_set_last_changed()` function as a whole says it is returning a `string` still:

```
@return string UNIX timestamp when the group was last changed.
```

Doesn't this need to be changed?

But if this is changed, will this be a back-compat breakage?

Reply:
You're right! What's the protocol/rules for changing the return type of a function?

Reply:
Typically it's not done 😬 

But when it needs to be done, it would probably require searching for existing themes/plugins that are using the return value to see if they would be affected by the change. If so, they'd need to update their implementation with a conditional check for whether it is WP>=6.9, for example.

Reply:
I changed it to `string`, let's stick to just a type fix.